### PR TITLE
Blobs are slimes. name them such.

### PR DESCRIPTION
**Pull request by esotericist**
#### Summary

SUMMARY: Bugfixes "Blobs are slimes. name them such."

#### Purpose of change

it has been a persistent source of confusion amongst players and contributors who read the design doc that the "blob" enemies you encounter in-game and the "blob" metaorganism behind the zombie-apocalypse are either directly related, or one and the same.

consensus amongst the devs in discussions on discord is that this is not desirable.

#### Describe the solution

renamed the "blob" enemies to appropriate variations on "slime", also renaming the "blob scrap" item to match.

also adjusted the species and factions.

#### Describe alternatives you've considered

i considered renaming the internal monster IDs as well, but that requires migration features not yet present.

abstractly there are other things that could potentially use renaming: monster special attacks, monster groups, etc. these all seemed more than a bit out of scope

#### Testing

loaded game, spawned some things, got a lack of errors and things seemed to hate each other that should've hated each other.

#### Additional context

fact of the day: the "blob scrap" item was already named `slime_scrap` internally.

### CODE REVIEW

**Comment by Xenomorph-III** [comment]
"other blobs seem to swarm around it..."

Blob -> slime.

File: data/json/monsters/slimes.json [hunk 1]
```diff
@@ -32,11 +32,11 @@
   {
     "id": "mon_blob_brain",
     "type": "MONSTER",
-    "name": { "str": "brain blob" },
+    "name": { "str": "brain slime" },
     "description": "A gigantic black blob of viscous goo, oozing across the ground like a glob of living oil.  Other blobs seem to swarm around it.",
```
Comment (anothersimulacrum) [inline data/json/monsters/slimes.json:36]:
```suggestion
    "description": "A gigantic black blob of viscous goo, oozing across the ground like a glob of living oil.  Other slimes seem to swarm around it.",
```

**Comment by meladath** [comment]
wait they ARE different? Oh wow. Yeah, its a good job this is being changed. My mind is blown. Maybe change the colour of the slime too? many places reference black blob/oil and I assumed it was all the same.

**Comment by nexusmrsep** [comment]
Aren't they the equivalent of living "cells" of The Blob? If not - what are they? if I lived all my life under this assumption, and it won't be easy for me to let go of that thought, if it's true.

**Comment by esotericist** [comment]
> wait they ARE different? Oh wow. Yeah, its a good job this is being changed. My mind is blown. Maybe change the colour of the slime too? many places reference black blob/oil and I assumed it was all the same.

i'm not really sure the color will make that big a difference?

> Aren't they the equivalent of living "cells" of The Blob? If not - what are they? if I lived all my life under this assumption, and it won't be easy for me to let go of that thought, if it's true.

they are a completely unrelated organism, just like other nether things. you'll notice in the monster definition they actually have `"NETHER"` as part of their species specification. they are absolutely not "cells" of the blob, according to what I have been told.

i dunno about where and when you formed your understanding (and obviously you aren't alone in this understanding, otherwise this change wouldn't be necessary), but i've been seeing kevin and other devs going "they're not related" literally since i got here (i.e. earlier last year), in all the venues i've seen them participate in.

**Comment by nexusmrsep** [comment]
> they are a completely unrelated organism

Holy smokes, I've been living a lie all this time.

Considering that The Blob  is also a a kind of nethereum denizen, I've never seen it as a potential discrepancy. Furthermore the living body of The Blob wouldn't be considered as a zombie faction as well. Sure, there wasn't any direct evidence to prove or disprove the relation between blobs and The Blob, but this definitely was a path to consider. And I actually loved the idea of them being more condensed/distilled form of a blob pathogen in water/zombies. That would suggest/imply that The Blob would also have a physical body that doesn't rely on hosts, and blobs as an intermediate form fits this theory like a glove. So - no wonder that I was deceived.

@kevingranade  you should definitely upgrade the design document on such matters. It's still too vague. I know that vagueness is an asset sometimes, but not every time, like in this example. Just pull in the conclusions of countless lore discussions and draw a line between what is and what is not.

**Comment by andrei8l** [comment]
There's a recipe for getting `slime_scrap` out of `meat_tainted` (`json/recipes/recipe_medsandchemicals.json:709`), and also the reverse for getting `meat_tainted` out of `slime_scrap` and `meat` (`json/recipes/recipe_food:5111`). If the slime monsters aren't supposed to be The Blob, those should be removed or changed as well.

I also vaguely remember a dev saying that The Blob is trying to change Earth into something more suitable and that the slime pits are definitely not the final form, but I can't track that down atm.

**Comment by esotericist** [comment]
@andrei8l please open a new issue. Merged PRs are not the right place for "this other thing needs doing", because by definition a merged PR is already closed

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/questions-about-spoilery-lore/26024/2

**Pull request merged**